requirements.in

```diff
@@ -47,4 +47,9 @@ djangorestframework-dataclasses>=1.3.1,<2
 djangorestframework-csv>=3,<4
 # As of 3.0 supports up to Django 4.2
 # Project: https://github.com/mjumbewu/django-rest-framework-csv
-# Changelog: https://github.com/mjumbewu/django-rest-framework-csv/releases
\ No newline at end of file
+# Changelog: https://github.com/mjumbewu/django-rest-framework-csv/releases
+
+sentry-sdk>=2,<3
```

Comment:
The docs say to install as `sentry-sdk[django]` for Django integration https://docs.sentry.io/platforms/python/integrations/django/#install , although in reality all that does is add a version constraint on the Django package https://github.com/getsentry/sentry-python/blob/master/setup.py#L56 it still might be a good idea in case they decide to deprecate the version of Django we're using in future?

Reply:
I've switched to this install method and separated out the commit into adding the package and the settings configs